projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-input/search-ai-input.component.ts

```diff
@@ -128,11 +128,16 @@ export class SearchAiInputComponent implements OnInit, OnDestroy {
     private userPreferencesService: UserPreferencesService,
     private translateService: TranslateService,
     private modalAiService: ModalAiService,
-    private route: ActivatedRoute
+    private route: ActivatedRoute,
+    private router: Router
   ) {}
 
   ngOnInit(): void {
-    if (this.searchTerm) {
+    const isKnowledgeRetrievalPage = this.router.url.startsWith('/knowledge-retrieval');
+
+    if (isKnowledgeRetrievalPage) {
+      this.queryControl.setValue('');
+    } else if (this.searchTerm) {
       this.queryControl.setValue(this.searchTerm);
     } else if (this.route.snapshot?.queryParams?.query?.length > 0) {
       this.queryControl.setValue(this.route.snapshot.queryParams.query);
```

Comment:
Is this case still needed? Wasn't it needed for a case when you are on results page and refresh it but now we don't display it on results page so maybe it's redundant?